x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -714,6 +714,61 @@ Gateway     | instances
 null        | null
 ;
 
+replace simple
+from employees | sort emp_no | limit 1 | eval name = replace(first_name, "Geo", "foo") | keep emp_no, name;
+
+emp_no:integer | name:keyword
+10001 | foorgi
+;
+
+replace text
+from hosts | where host == "epsilon" | limit 1 | eval name = replace(host, "ep", "") | keep name;
+
+name:keyword
+silon
+;
+
+replace complex
+from employees | where emp_no <= 10010 | eval f_l = replace(replace(substring(last_name, 1, 20), "al", "AB"), "a", "Z")  | keep emp_no, last_name, f_l;
```

Comment:
Please add a `| sort emp_no` somewhere in these two queries, otherwise they'll randomly fail in multi-node